src/server/pkg/storage/gc/util.go

```diff
@@ -0,0 +1,152 @@
+package gc
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/jinzhu/gorm"
+	"github.com/lib/pq"
+	"github.com/pachyderm/pachyderm/src/server/pkg/backoff"
+	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	defaultPostgresHost = "localhost"
+	defaultPostgresPort = 32228
+)
+
+// NewLocalDB creates a local database client.
+// (bryce) this should be somewhere else.
+// probably a db package similar to obj.
```

Comment:
Agree with this comment, `src/server/pkg/db` seems like the right place.

Reply:
So, can we punt on this, or do you feel it needs to be addressed in this PR? I figured this would be a part of the global id stuff because that is going to deal more with setting up / using the postgres client in pfs.

Reply:
No need to address in this PR, I just saw the comment and figured I'd agree with it :)